frontend/src/App.css

```diff
@@ -522,16 +542,25 @@ header {
     justify-content: center;
   }
 
+  .search-input-wrapper {
+    position: sticky;
+    top: 0;
+    background: #292929;
+    box-shadow: 0 5px 3px #292929;
+  }
```

Comment:
can you please add a z-index for this selector because the resources are overlapping with the search bar at the moment. See image:
![image](https://user-images.githubusercontent.com/54069197/192165222-492f1d41-6da8-4c6f-a79a-53dcaf87ab78.png)
Make sure the search bar and search suggestions are over the top resources with a dark grey background

Reply:
Oh...It worked in my case as you can see on my screenshot. ok, I will add the z-index.

Reply:
I added the z-index and it's now working properly. ✌